### ISSUE TITLE
Expand VSCode variables for config file

### ISSUE THREAD

**Issue by aviramha** [metalbear-co/mirrord-vscode#30]
`MIRRORD_CONFIG_FILE` should be able to accept `${var}` so we can do for example `"${workspaceFolder}/ip-visit-counter/mirrord.json"`

**Comment by Razz4780** [comment]
If the path is relative, it is already resolved relative to the workspace folder. Do we want this to be more explicit?

**Comment by aviramha** [comment]
> If the path is relative, it is already resolved relative to the workspace folder. Do we want this to be more explicit?

No. If you try using `${workspaceFolder}` as value for `MIRRORD_CONFIG_FILE` it won't work, since the `${..}` is passed as is to the mirrord configuration, which doesn't have the value of `$workspaceFolder`

**Comment by eyalb181** [comment]
@Razz4780 It's resolved relative to the directory mirrord runs from, which can be configured to not be the workspace folder.
